### PR TITLE
README: Update with correct install snap command

### PR DESCRIPTION
**Pull request by nitkon**
Improve README and update with 
correct install snap command.

Fixes:  #469

Signed-off-by: Nitesh Konkar niteshkonkar@in.ibm.com

### CODE REVIEW

File: snap/README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ This directory contains the resources needed to build the Kata Containers
 
 ## Initial setup
 
-*Ubuntu 18.04*
-
+Kata Containers can be installed in any Linux distribution that supports
+[snapd](https://docs.snapcraft.io/installing-snapd).
 ```sh
 $ sudo apt-get install -y snapd snapcraft
```
Comment (devimc) [inline snap/README.md]: but here we are using `apt-get`, I think you should mention that for example the following command is to install snapd in ubuntu

File: snap/README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ $ sudo apt-get install -y snapd snapcraft
 You can install the Kata Containers snap from the [snapcraft store][8] or by running the following command:
 
 ```sh
-$ sudo snap install kata-containers
+$ sudo snap install kata-containers --classic
```
Comment (devimc) [inline snap/README.md]: good catch!, thanks

File: snap/README.md (hunk 1 above)
Comment (nitkon) [reply snap/README.md]: @devimc : Fixed.

**Pull request merged**